src/index.html

```diff
@@ -2,12 +2,57 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars--0">
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+      </div>
+      <div class="stars--1">
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+      </div>
+      <div class="stars--2">
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+      </div>
+      <div class="stars--3">
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+        <div class="stars--star"></div>
+      </div>
+      <div class="stars--4">
```

Comment:
```suggestion
      <div class="stars stars--4">
```
combine 2 classes in each starts row 😉